src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_lease.py

```diff
@@ -120,10 +121,8 @@ def _read(self):
             valid_lifetime = splits[3]
             lease_end = splits[4]
 
-            if mac_address not in fdb_info:
-                syslog.syslog(syslog.LOG_WARNING, "Cannot not find {} in fdb table".format(mac_address))
-                continue
-            new_key = "{}|{}".format(fdb_info[mac_address], mac_address)
+            vlan_name = fdb_info[mac_address] if mac_address in fdb_info else DHCP_UNKNOWN_INTERFACE
```

Comment:
Better to use dhcp_interface as var name

Reply:
Yes, updated.